docs/content/basic/authenticating-oauth.md

```diff
@@ -4,256 +4,703 @@ lang: en
+### Installing your App
 
-* **Open Slack**: After users **Open Slack**, and here after as your app processes events from Slack, your provided `installationStore`'s `fetchInstallation` and `storeInstallation` handlers will execute. See the **Installation Object** section below for more detail on arguments passed to those handlers.  
+Bolt for JavaScript provides an **Install Path** `/slack/install`
+out-of-the-box. This endpoint returns a simple static page that includes an
+`Add to Slack` button that links to a generated authorization URL for your app.
+This has the right scopes, a valid `state`, the works.
 
-* If you need additional authorizations (user tokens) from users inside a team when your app is already installed, or have a reason to dynamically generate an install URL, manually instantiate an `ExpressReceiver`, assign the instance to a variable named `receiver`, and then call `receiver.installer.generateInstallUrl()`. Read more about `generateInstallUrl()` in the [OAuth docs](https://slack.dev/node-slack-sdk/oauth#generating-an-installation-url).
+An app hosted at _www.example.com_ will serve the install page at
+_www.example.com/slack/install_ but this path can be changed with
+`installerOptions.installPath`. Rendering a webpage before the authorization URL
+is also optional and can be skipped using `installerOptions.directInstall`.
 
-:::info 
+Inspect this [example app][direct-install] and snippet below:
 
-Bolt for JavaScript does not support OAuth for [custom receivers](/concepts/receiver). If you're implementing a custom receiver, you can use our [Slack OAuth library](https://slack.dev/node-slack-sdk/oauth#slack-oauth), which is what Bolt for JavaScript uses under the hood.
+```javascript
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  // ...
+  installerOptions: {
+    // highlight-start
+    directInstall: true,
+    installPath: "/slack/installations", // www.example.com/slack/installations
+    // highlight-end
+  },
+});
+```
+
+#### Add to Slack button
+
+The [default][add-to-slack] `Add to Slack` button initiates the OAuth process
+with Slack using a generated installation URL. If customizations are wanted to
+this page, changes can be made using
+[`installerOptions.renderHtmlForInstallPath`][installation-page] and the
+generated installation URL:
+
+```javascript
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  // ...
+  installerOptions: {
+    // highlight-start
+    renderHtmlForInstallPath: (addToSlackUrl) => {
+      return `<a href="${addToSlackUrl}">Add to Slack</a>`;
+    },
+    // highlight-end
+  },
+});
+```
+
+We do recommend using the provided [button generator][oauth-v2] when formatting
+links to the authorization page!
+
+:::note
+
+Authorization requests with changed or additional scops require
```

Comment:
```suggestion
Authorization requests with changed or additional scopes require
```

Reply:
Sad because I actually needed `scops` to land in this PR 🪐 🔭